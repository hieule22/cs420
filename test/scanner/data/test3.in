thistestcasecontainseverycategoryoflexeme
program procedure int bool begin end if then else while print not
;:,:=()
=<>>=<<=>
+-or
and*/
programnotakeyword
integer
andor
orand
000001
123456789
