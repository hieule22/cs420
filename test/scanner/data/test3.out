IDENTIFIER thistestcasecontainseverycategoryoflexeme
PROGRAM
PROCEDURE
INT
BOOL
BEGIN
END
IF
THEN
ELSE
WHILE
PRINT
NOT
SEMICOLON
COLON
COMMA
ASSIGNMENT
OPENBRACKET
CLOSEBRACKET
EQUAL
NOTEQUAL
GREATEROREQUAL
LESSTHAN
LESSOREQUAL
GREATERTHAN
ADD
SUBTRACT
OR
AND
MULTIPLY
DIVIDE
IDENTIFIER programnotakeyword
IDENTIFIER integer
IDENTIFIER andor
IDENTIFIER orand
NUMBER 000001
NUMBER 123456789
ENDOFFILE